projects/plugins/backup/.gitattributes

```diff
@@ -1,13 +1,24 @@
-# Files not needed to be distributed in the package.
-.gitattributes    export-ignore
-.github/          export-ignore
-package.json      export-ignore
+# Files to include in mirror repo: Automattic/jetpack-backup-plugin
+build/**                      production-include
+jetpack-backup.php            production-include
+src/**                        production-include
```

Comment:
These line should be unnecessary. `production-include` is basically for overriding `.gitignore` but these aren't ignored.

```suggestion
```